test/Interop/Cxx/destructor/IRGen.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir | %FileCheck %s --check-prefix=CHECK
```

Comment:
Could you rename this test and other tests to follow the naming pattern of other tests in `test/Interop`? `basic-irgen.swift`